sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleTranslationWithGlossaries.cs

```diff
@@ -0,0 +1,80 @@
+﻿// <copyright file="SampleTranslationWithGlossaries.cs" company="Microsoft Corporation">
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+
+using Azure;
+using Azure.AI.Translation.Document;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+/**
+FILE: SampleTranslationWithGlossaries.cs
+DESCRIPTION:
+    This sample demonstrates how to create a translation job and apply custom glossaries to the translation.
+USAGE:
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+    3) AZURE_SOURCE_CONTAINER_URL - the container SAS URL to your source container which has the documents
+        to be translated.
+    4) AZURE_TARGET_CONTAINER_URL - the container SAS URL to your target container where the translated documents
+        will be written.
+    5) AZURE_TRANSLATION_GLOSSARY_URL - the SAS URL to your glossary file
+**/
+
+namespace DocumentTranslatorSamples
+{
+    class SampleTranslationWithGlossaries
```

Comment:
in our multiple inputs sample we added a glossary. Do you think it is still worth it having as a separate sample?
If so, could you adapt it to follow more of our .NET samples, like here? 
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs#L38

Reply:
Removed (Duplicate)